csrc/scheduler/matmul.cpp

```diff
@@ -437,6 +463,102 @@ void scheduleProlog(TensorView* shared_mem_tv, const MatmulParams& params) {
           .propagateParallelType());
 }
 
+void schedule_output_tensor(
+    TensorView* c,
+    int warp_tile_m,
+    int instruction_tile_m) {
+  // [a,b,128,128]
+  // Distribute warp tile:
+  c->split(-2, warp_tile_m);
+  //[a,b,128/wm, wm, 128]
+
+  c->split(-2, instruction_tile_m);
+  //[a,b,128/wm, wm/im, im, 128]
+  c->split(-2, 2);
+  //[a,b,128/wm, wm/im, im/2, 2, 128]
+
+  c->split(-1, 4);
+  //[a,b,128/wm, wm/im, im/2, 2, 128/4, 4]
+  // 0,1, 2,     3,      4,   5,  6 ,   7
+  c->reorder({{2, 3}, {3, 2}});
+  //[a,b,wm/im, 128/wm, im/2, 2, 128/4, 4]
+  int axis = 0;
+  c->axis(axis++)->parallelize(ParallelType::BIDx);
+  c->axis(axis++)->parallelize(ParallelType::BIDy);
+  c->axis(axis++)->parallelize(ParallelType::Serial);
+  c->axis(axis++)->parallelize(ParallelType::TIDz);
+  c->axis(axis++)->parallelize(ParallelType::Serial);
+  c->axis(axis++)->parallelize(ParallelType::TIDy);
+  c->axis(axis++)->parallelize(ParallelType::TIDx);
+  c->axis(axis++)->parallelize(ParallelType::Vectorize);
+}
+
+void schedule_epilogue_tensor(
```

Comment:
```suggestion
void scheduleEpilogueTensor(
```

Reply:
Is this copy-pasted from here?
https://github.com/NVIDIA/Fuser/blob/7bd98cd81ef75e364948cbe3674de27787033055/csrc/scheduler/mma_utils.cpp#L1054
Can we not copy-paste the code, and use transform propagator?
I am thinking that, if you can call 
```C++
c_smem->applyMmaSwizzle(
      mma_builder.operand(MmaOptions::Operand::Accumulator).build());
```
and propagate this to mma_result.

Reply:
If we directly propagate mma output tensor to epilogue tensor, the swizzle information is lost and leads to bank conflict. I changed to schedule the epiloge tensor by reusing `scheduleWarpTileWithNoReduction` and `applyMmaSwizzle`.

Reply:
How about schedule the epilogue tensor and propagate to the mma output?